docs/source/notes/mkldnn.rst

```diff
@@ -0,0 +1,101 @@
+.. meta::
+   :description: A guide to torch.backends.mkldnn, a PyTorch backend to run MKLDNN operations
+   :keywords: optimize PyTorch, MKLDNN
+
+.. _mkldnn_backend:
+
+MKLDNN backend
```

Comment:
For this page and the updated cuda one above, could you clarify what's the message you're trying to get across here?
All these flags should already be described in the list of APIs in the backends.rst side already.

We should only cover in the notes.rst specifics of the platform or things that don't fit in the regular doc discussion.

Reply:
For cuda one, we wish to updated the usage there since we changed the API and there are descriptions of old API there.
For mkldnn one, we wish to have some similar documents (MKLDNN+BF16 in mkldnn.rst) with CUDA tf32 in cuda.rst.